.github/workflows/release.yml

```diff
@@ -235,25 +241,33 @@ jobs:
         with:
           name: deb
 
+      - uses: docker/setup-qemu-action@v1
+
       - uses: docker/setup-buildx-action@v1
 
       - name: generate dockerfile
         run: |
           echo 'FROM ${{ matrix.distro }}
-          COPY tyk-pump*_amd64.deb /tyk-pump.deb
+          ARG TARGETARCH
+          COPY tyk-pump*_${TARGETARCH}.deb /tyk-pump.deb
+          RUN apt-get update && apt-get install -y curl
+
+          RUN curl -fsSL https://packagecloud.io/install/repositories/tyk/tyk-pump/script.deb.sh | bash && apt-get install -y tyk-pump=1.2.0
```

Comment:
Is it ok that 1.2.0 is hard coded here?

Reply:
So this is the version that we will upgrade from. It is chosen as the last stable release before goreleaser landed. It is defined for all the repos at https://github.com/TykTechnologies/tyk-ci/blob/master/wf-gen/header.m4#L12

Reply:
So with every new major release we should not forget to update it? If so, just ensure that it is included in some checklist.

Reply:
I do not think it is _necessary_ to update this version for every update. Another option is to have an array of old versions and test every version.

Reply:
Yeah, array does make sense, to spot issues between bigger upgrades.

Reply:
As for adding it to the checklist, would you please point me to the checklist that needs to be added to or the person who runs the checklist?

Reply:
Can we push making it array for the future? Right now we don't have an array.